soc/arm/silabs_exx32/common/soc_power.c

```diff
@@ -63,3 +151,69 @@ __weak void pm_state_exit_post_ops(enum pm_state state, uint8_t substate_id)
 	ARG_UNUSED(state);
 	ARG_UNUSED(substate_id);
 }
+
+#ifdef CONFIG_SOC_GECKO_DEV_INIT
+
+/* Function to check any ready threads are scheduled */
+bool ConfirmSleepModeStatus(void)
```

Comment:
Why the CamelCase?

Reply:
I've refactored the code and this piece of code is no longer needed.